app/src/main/java/org/mozilla/fenix/settings/DefaultBrowserSettingsFragment.kt

```diff
@@ -89,6 +89,8 @@ class DefaultBrowserSettingsFragment : PreferenceFragmentCompat() {
             isChecked = context.settings().openLinksInAPrivateTab
```

Comment:
No action needed: I noticed the `Browsers` initializer is pretty slow (20ms on my Pixel 2) and we're fetching it three times in this method: `updateSwitch`, the explicit usage, and in `unsetOpenLinks...`. Optimizing `DefaultBrowserSettingsFragment` is not our priority at the moment but I wanted to raise it.

The `Browsers` impl may do some internal caching that might mitigate the repeated access though.